themes/V3/Blank/style.less

```diff
@@ -481,4 +480,13 @@ body { counter-reset : page-numbers; }
 	&:nth-child(even) {
 		.pageNumber { left : 30px; }
 	}
-}
+
+	.resetCounting {
+		counter-set : page-numbers 1;
+	}
```

Comment:
wouldn't we want the next page to be 1? if so, we would set this one to 0. Not sure if i missed something.

Reply:
No. Reset applies to the current page.

Reply:
Sorry, i was wrongly assuming the resseting page would also be skipped, don't know where i got that assumption, you are right.